### ISSUE TITLE
Display time to integrate

### ISSUE THREAD

**Issue by asim** [m3o/m3o#148]
Most of the time when it comes to integrate APIs, it's really unclear how long its going to take. We don't know the complexity, we can't guarantee it'll be done in a short amount of time. With simpler APIs, it's clear that we can scope a lot of this and potentially provide guarantees around that, even display it to people to understand how quickly they can get up and running.

**Comment by mawoka-myblock** [comment]
> 

Maybe a community board or something like this could be helpful, to show the status, etc.

**Comment by asim** [comment]
Like comments on the site?

**Comment by mawoka-myblock** [comment]
> Like comments on the site?

Nah, not really. I have seen a board, where users could upvote ideas and submit them. Not really comments. Let me see if I can find the website!

**Comment by asim** [comment]
Oh I see. I built a thing like that https://distributed.m3o.app

**Comment by mawoka-myblock** [comment]
> Oh I see. I built a thing like that [distributed.m3o.app](https://distributed.m3o.app)

Yea, maybe but a bit more obviously! And maybe link the login with the account of m3o.com
